code/_onclick/ai.dm

```diff
@@ -188,33 +188,33 @@
 /obj/machinery/door/airlock/AIAltShiftClick(mob/user)  // Sets/Unsets Emergency Access Override
 	if(!ai_control_check(user))
 		return
-	toggle_emergency_status(user)
+	addtimer(CALLBACK(src, /obj/machinery/door/airlock/.proc/toggle_emergency_status, user), 5 SECONDS)
```

Comment:
This will feel like the game is not responding since they are not getting feedback. A `do_after` will be better here